dts/bindings/bluetooth/sifli,sf32lb-mailbox.yaml

```diff
@@ -0,0 +1,16 @@
+# Mailbox Bluetooth HCI device
+
+description: |
```

Comment:
mailbox binding should be in mbox, it just seems to be a mailbox type device to signal data exchanged via a shared ram buffer. It also seems like what is in hal ipc_hw.c should be a zephyr mbox driver, which allows to implement an interface for such kind of peripherals.